lib/y2lan_restart_common.pm

```diff
@@ -150,12 +150,14 @@ sub check_network_status {
     assert_script_run 'cat journal.log';                              # print journal.log
     my $is_empty = script_run('[ -s journal.log ]');
     if ($is_empty) {
-        assert_equals($expected_status, 'no_restart', "Network restart didn't occur, even though it was expected.");
+        assert_equals($expected_status, 'no_restart', "Do not expect Network restart/reload. journalctl cannot find events");
```

Comment:
So we get here when journal.log is empty which means no restart happened . 3d parameter of `assert_equals` is error message which would be displayed when first two params **did not match** . Second param is `no_restart` so to get error `$expected_status` should `restart` or `reload` . So error message does not fit situation when we will see it

Reply:
@asmorodskyi could you review when ever the time. thanks for catching this.